bishnu-suyel/Druid-Final-Project
# Tdfp11 - Test Media module and fetching images

### In this PR 

- installed a Media Module to upload and manage images
- added _fetchImages_ function in api.jsx to fetch all images from the jsonapi endpoint
- rendered fetched images in _Services.jsx_

### To test

1. `git pull`
2. import database `lando db-import dump.sql`
3. import configurations `lando drush cim`
4. go to 'Content' -> Media tab 
5. upload some images
6. fetch and render images in your React component
7. make sure to update `API-URL ` to your localhost address.
